### ISSUE TITLE
[Bug] the softmax gave wrong inference results

### ISSUE THREAD

**Issue by jikechao** [apache/tvm#14721]
### Expected behavior

the predicted result in tvm should be the same as keras.

### Actual behavior

inconsistent inference results between keras and tvm.
![image](https://user-images.githubusercontent.com/29506758/234540443-2ba87a18-8a00-4bb6-85c8-06cf0a4b44b2.png)

### Steps to reproduce

```
import tvm
import tvm.relay as relay
import numpy as np
from tensorflow import keras
from tensorflow.keras import layers, models


input_shape = (1, 2, 2, 3)
input_data = np.random.random(input_shape)
x = layers.Input(shape=input_shape[1:], dtype='float32')

layer = keras.layers.Softmax()
layer.set_weights(layer.get_weights())

y = layer(x)
model = models.Model(x, y)
# print(model.summary())
res_keras = model.predict(input_data)

shape_dict = {'input_1': input_shape}
mod, params = relay.frontend.from_keras(model, shape_dict)

with tvm.transform.PassContext(opt_level=3):
    model = relay.build_module.create_executor("graph", mod, tvm.cpu(0), 'llvm', params).evaluate()

res_tvm = model(tvm.nd.array(input_data.astype('float32'))).numpy()

print('keras infer result:', res_keras)
print('tvm infer result:', res_tvm)
np.testing.assert_allclose(res_keras, res_tvm, atol=1e-3, rtol=1e-3)
```

### Triage
*  frontend:keras

**Comment by dmilosevic252** [comment]
The error is in the line `mod, params = relay.frontend.from_keras(model, shape_dict)`, it should be `mod, params = relay.frontend.from_keras(model, shape_dict,"NHWC")`. I tested the code with this line and the results match.
